code/API_definitions/geofencing-subscriptions.yaml

```diff
@@ -57,6 +57,14 @@ info:
       Developers may provide a callback URL on which notifications regarding geofencing can be received from the service provider.
       If an event occurs the application will send events to the provided webhook - 'notificationUrl'._
 
+    ### Authorization and authentication
```

Comment:
```suggestion
    # Authorization and authentication
```

Reply:
I am okay with that, but should we update this as well in Commonalities? 

https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#mandatory-template-for-infodescription-in-camara-api-specs

Reply:
@fernandopradocabrillo Probable good to raise an issue to ICM team to propose the correction - wdyt?